shuffle.py

```diff
@@ -91,6 +91,10 @@ def make_conf(params):
     else:
         append_require(params, conf, 'protobuf')
 
+    if params.get('ideep') is not None:
```

Comment:
We should exclude iDeep4Py combination if not the Python is not (2.7.5+, 3.5.2+, or 3.6.0+).
(Maybe this depends on https://github.com/chainer/chainer-test/pull/383)

Reply:
Fixed.